### ISSUE TITLE
Why are mini and mini_valid in tnas.py sampled from the same dataset.

### ISSUE THREAD

**Issue by euminds** [dongzelian/T-NAS#7]
# batch_size here means total episode number
    mini = MiniImagenet(args.data_path, mode='train', n_way=args.n_way, k_shot=args.k_spt,
                        k_query=args.k_qry,
                        batch_size=args.batch_size, resize=args.img_size)
    mini_valid = MiniImagenet(args.data_path, mode='train', n_way=args.n_way, k_shot=args.k_spt,
                        k_query=args.k_qry,
                        batch_size=args.test_batch_size, resize=args.img_size)

    train_loader = DataLoader(mini, args.meta_batch_size, shuffle=True, num_workers=args.num_workers, pin_memory=True)
    valid_loader = DataLoader(mini_valid, args.meta_test_batch_size, shuffle=True, num_workers=args.num_workers, pin_memory=True)
mini and mini_valid are  sampled from train dataset, but mini is used for meta_train and mini_valid is used for meta_test. So if mini_valid is a part of mini, it is unfair. And I divide the train dataset into 80% and 20%. mini is sampled from 80% and mini_valid is sampled from 20%. Finally acc by running train_tnas.sh is 0.57  but the original acc by runing train_tnas.sh is 0.60.
